Source/Widgets/CesiumWidget/CesiumWidget.js

```diff
@@ -144,6 +144,7 @@ function configureCameraFrustum(widget) {
  * @param {Boolean} [options.shadows=false] Determines if shadows are cast by light sources.
  * @param {ShadowMode} [options.terrainShadows=ShadowMode.RECEIVE_ONLY] Determines if the terrain casts or receives shadows from light sources.
  * @param {MapMode2D} [options.mapMode2D=MapMode2D.INFINITE_SCROLL] Determines if the 2D map is rotatable or can be scrolled infinitely in the horizontal direction.
+ * @param {Boolean} [options.blurActiveElementOnCanvasFocus=true] If true, the the active element will blur when the viewer's canvas is clicked.
```

Comment:
> I don't believe this defaults to `true` when using `CesiumWidget` directly. I would expect that `defaultValue` would set the default in this function rather than a level up in `Viewer`. All `Viewer` needs to do is pass through the value to `CesiumWidget`, whether its defined or not.

That resaonable. But why do we specify default values in the viewer's constructor's jsdoc if we dont actually pass them?
For example mapMode2D defaults to INFINITE_SCROLL, even though this paramater only defaults to it when constructing Scene.

Reply:
> Lastly, I think it would help if we briefly explain in the documentation under what circumstances this would be useful in.

Do you mean inline comments?

Reply:
We should pass the value of the option from `Viewer` to `Widget`. However, if that value is `undefined`, `Widget` then used the default.

Reply:
> Do you mean inline comments?

I was thinking a sentence or so in the reference doc comment, so that users reading the documentation who may not immediately know what the option is could have bit more context.